src/main/java/uk/nickbdyer/chatserver/ChatServer.java

```diff
@@ -18,15 +18,7 @@ public ChatServer(ServerSocket serverSocket, OutputStream receivedMessage) {
     public void listen() {
```

Comment:
Any ideas on a better name for this method? It listens for connections, and then processes any text it gets from those connections.